workflow/snakemake_rules/fetch_sequences.smk

```diff
@@ -104,12 +104,13 @@ if config.get("s3_dst") and config.get("s3_src"):
         params:
             file_on_s3_dst=f"{config['s3_dst']}/{database}.ndjson.zst",
             file_on_s3_src=f"{config['s3_src']}/{database}.ndjson.zst",
+            lines = config["subsample"].get("main_ndjson", 0)
```

Comment:
I would change all instances of `config["subsample"]` to `config.get("subsample", {})` since the `subsample` param is optional for this pipeline. 

Currently the automated builds would fail with a `KeyError` because they do not have `subsample` in their configs.

Reply:
Ah, I see this got updated in a later commit. Please fix up into the original commit before merging.